model/src/main/proto/preprocess_protos.bzl

```diff
@@ -0,0 +1,21 @@
+
+def process(name, src):
+  """
+  Formats the import statement of the src file to contain a full path to the file
```

Comment:
This should describe what is happening & how the macro should be used, not necessarily what the implementation is doing.

Reply:
Okay, makes sense! How does this sound: 
"Full path to .proto file imports are needed when building with Bazel but breaks Gradle builds.
  Use this macro to preprocess and build .proto files with import statements using Bazel"

Reply:
I think your current documentation sounds good.